core/procedures.js

```diff
@@ -342,7 +342,7 @@ Blockly.Procedures.getDefinition = function(name, workspace) {
  * @private
  */
 Blockly.Procedures.editProcedureCallback_ = function(block) {
-  if (block.type == 'procedures_defnoreturn') {
+  if (block.type == Blockly.BlockSvg.DEFINE_BLOCK_TYPE) {
```

Comment:
Seems slightly strange that the `PROCEDURES_CALL_BLOCK_TYPE` is defined on blockly, and `DEFINE_BLOCK_TYPE` is defined on BlockSvg?

Reply:
Fixed, which also fixed the tests.